Ginger/GingerCoreCommon/VariablesLib/VariableSelectionList.cs

```diff
@@ -48,9 +48,24 @@ public string OptionalValues
 
         [IsSerializedForLocalRepository]
         public ObservableList<OptionalValue> OptionalValuesList = new ObservableList<OptionalValue>();
-
+        [IsSerializedForLocalRepository]
         public string SelectedValue { set { Value = value; OnPropertyChanged(nameof(SelectedValue)); } get { return Value; } }
 
+        private string mValue;
+
+        public override string Value
```

Comment:
we decided to override and serialize the value field for selection list variable. no?

If we serialize value then selected value serialize is not needed